distributed/shuffle/_rechunk.py

```diff
@@ -441,47 +447,98 @@ def _() -> dict[str, tuple[NDIndex, bytes]]:
         return self.run_id
 
     async def get_output_partition(
-        self, partition_id: NDIndex, key: str, meta: pd.DataFrame | None = None
+        self, partition_id: NDIndex, key: str, **kwargs: Any
```

Comment:
Is this switch to `kwargs` future-proofing?

Reply:
Future-proofing and making it completely transparent to the plugin.